src/Common/src/Interop/Unix/System.Native/Interop.PathConf.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Sys
+    {
+        internal static int DEFAULT_PC_NAME_MAX = 255;
```

Comment:
The only reason the Max Path default value needed to be pushed into the PAL was because the PATH_MAX value changes between OS X, Linux, and other *nix flavors that we support. The PC Name Max does not look to have different default values (from our original implementation) so it does not need to be pushed down into the PAL and we can use the platform-agnostic 255 value